docs/04-operation-guides/operations/obsv-03-enable-istio-access-logs.md

```diff
@@ -45,4 +58,20 @@ spec:
   selector:
     matchLabels:
       service.istio.io/canonical-name: {YOUR_LABEL}
+```
+
+### Configure Istio access logs for a specific gateway
+
+Instead of enabling the access logs for all the individual proxies of the workloads you have, you could instead enable the logs for the proxy used by the related Istio ingress gateway.
```

Comment:
```suggestion
Instead of enabling the access logs for all the individual proxies of the workloads you have, you can enable the logs for the proxy used by the related Istio ingress gateway:
```